learning-another-language/summary-resources-and-next-steps.md

```diff
@@ -0,0 +1,85 @@
+- We indent inner blocks of code
+- JavaScript shares the following things in common with Python: the mod operator, checking equality with `==`, and `return` statements
+
+## Resources
+
+Finding the right resources comes with practice and time.
+
+When we need to research something, it might be to answer one of these questions:
+
+1. How do we _do_ something?
+1. What _is_ the definition of something? How does something work? What do I need to know?
+
+### Suggested Starting Places
+
+When we need to learn how to _do_ something, as a suggestion, we can focus on:
+
+- The syntax to accomplish it
+- The expected result
+- Finding variations of different syntaxes for different situations
+- Tutorials to achieve what we need
+- StackOverflow answers
+- Documentation and reference material about the syntax
+
+When learning what _is_ something and how something works, we can focus on:
+
+- Learning how something fits in with JavaScript
+- Videos or diagrams explaining the process
+- Blogs
+- Documentation and reference material about the concepts
+
+### Suggested Ways to Evaluate
+
+For any given resource, we can evaluate based on:
+
+| Criteria                    | Notes                                                                                                                                                                      |
+| --------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Date written/published      | The older that a resource is, the more outdated it is with JavaScript _syntax_. Concepts and patterns change every few years, too.                                         |
+| Context                     | Who wrote this? Why did they write it? What biases do they carry? What is the intended purpose of this material?                                                           |
+| Source                      | It's helpful to distinguish between an independent developer or a product who authored a resource. Both kinds of authors genuinely want to better the developer community. |
+| Depth                       | Sometimes we don't need very deep explanations for some syntax. Sometimes we need deep explanations to understand a fundamental concept.                                   |
+| Number and Kind of Examples | Depends on what we're looking for!                                                                                                                                         |
+
+### !callout-info
+
+## ECMAScript 2015, aka ES6
+
+The JavaScript language went through a major revision in 2015. It introduced _dozens_ of new syntaxes, features, and patterns. There is usually a large difference in code style patterns between JavaScript before 2015 and after 2015.
```

Comment:
avoids "syntaxes"

we don't count in dozens enough... for those times when tens is too few and hundreds is too many. 🙂 

```suggestion
The JavaScript language went through a major revision in 2015. It introduced _dozens_ of new syntax elements, features, and patterns. There is usually a large difference in code style patterns between JavaScript before 2015 and after 2015.
```